packages/react-components/src/components/TextFieldWithMention/mentionTagUtils.ts

```diff
@@ -342,13 +347,18 @@ export const updateHTML = (props: UpdateHTMLProps): { updatedHTML: string; updat
     if (startIndex <= tag.plainTextBeginIndex) {
       // change start is before the open tag
       // Math.max(lastProcessedPlainTextTagEndIndex, startIndex) is used as startIndex may not be in [[previous tag].plainTextEndIndex - tag.plainTextBeginIndex] range
-      const startChangeDiff = tag.plainTextBeginIndex - Math.max(lastProcessedPlainTextTagEndIndex, startIndex);
+      const startChangeDiff = getChangeDiff(
```

Comment:
Is it possible to use a separate parameter (like `mentionTagLength`), calculate it once for all subtags and reuse instead of calculating it for every tag?

Reply:
Talked to Viktoriia async. We'll keep this as is since the calculation is depends on different startIndex and endIndex. Also, for each tag, we're calculation different portion of the string.